terraform/Dockerfile

```diff
@@ -1,13 +1,28 @@
-FROM alpine:3.3
+FROM golang:1.7.1-alpine
 
 ENV PATH $PATH:/usr/local/bin
-ENV TERRAFORM_VER 0.7.3
-ENV TERRAFORM_SUM 4e985f222ec99616e8c730d737c9b400f9d73bf0c436661ec888b2406d3a6f39
-ENV TERRAFORM_ZIP terraform_${TERRAFORM_VER}_linux_amd64.zip
+ENV TERRAFORM_VER 0.7.5
+ENV TERRAFORM_TAG datadog_dashboard_type_style
+ENV DATADOG_API 131428567_dashboard_options
```

Comment:
That's a little bit confusing naming. It thought it is Datadog api key or sth.  Is it actually a tag or branch?